tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -130,42 +142,178 @@ private static ArgumentParser argParser() {
                 .metavar("THROUGHPUT")
                 .help("throttle maximum message throughput to *approximately* THROUGHPUT messages/sec");
 
+        parser.addArgument("--num-threads")
+                .action(store())
+                .required(true)
+                .type(Integer.class)
+                .metavar("NUM-THREADS")
+                .dest("numThreads")
+                .help("The value bound of the random integers in message payload to simulate different compression ratio.");
```

Comment:
It seems that you have used doc of `value-bound` for `num-threads` by mistake. The comment above should apply to doc for `value-bound`.